spikewidgets/widgets/unitwaveformswidget/unitwaveformswidget.py

```diff
@@ -89,6 +100,8 @@ def plot_unit_templates(recording, sorting, channel_ids=None, unit_ids=None, ms_
     channel_locs: bool
         If True, channel locations are used to display the waveforms.
         If False, waveforms are displayed in vertical order. (default)
+    radius: float
+        If not None, all channels within a circle around the peak waveform will be displayed. Ignores max_spikes_per_unit.
```

Comment:
can you add add new args in the docstrings?

Reply:
done